### PR TITLE
Provide guidance on configuring Serilog and how it impacts .NET Core & Azure Functions

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Provide guidance on configuring Serilog and how it impacts .NET Core & Azure Functions.

Relates to #118

### CODE REVIEW

File: docs/preview/guidance/use-with-dotnet-and-functions.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+If you cannot use `UseSerilog`, you can still configure it by using `AddSerilog` as a logging provider; but we recommend removing all other providers with `loggingBuilder.ClearProviders()` so that they don't interfer.
+
+## Setting up Serilog with Azure Functions
+
+Using Serilog with Azure Functions requires some guidance and we've made it a bit easier to use.
+
+Before we get started, install our NuGet package for Azure Functions:
+
+```
+PM > TODO @stijnmoreels
```
Comment (tomkerkhove) [inline docs/preview/guidance/use-with-dotnet-and-functions.md:26]: @stijnmoreels  TODO in your PR where it's being added, not have to change now

File: docs/preview/guidance/use-with-dotnet-and-functions.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+```
+
+Once that is done, you can configure Serilog during startup as following:
+
+```csharp
+using Arcus.TODO.@stijnmoreels;
```
Comment (tomkerkhove) [inline docs/preview/guidance/use-with-dotnet-and-functions.md:32]: @stijnmoreels  TODO in your PR where it's being added, not have to change now

Comment (stijnmoreels) [inline docs/preview/guidance/use-with-dotnet-and-functions.md:32]: It's an complete example, I'm jus wondering if it's not "too much". Maybe having a little less enrichers levels, ... IDK. It seems a bit "noise" for what we want to talk about here?

Comment (stijnmoreels) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: Or maybe with a code comment to highlight the parts is also an option.

File: docs/preview/guidance/use-with-dotnet-and-functions.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:26]: Yes, of course.

File: docs/preview/guidance/use-with-dotnet-and-functions.md (hunk 2 above)
Comment (stijnmoreels) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: Yes, of course.

Comment (tomkerkhove) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: That's fine for me, do you want to get rid of the code and use // Your logic here?

Comment (stijnmoreels) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: Was just a suggestion. Not sure how and what lines we could remove to keep it as simple as possible. Maybe the enrichers can be removed?

Comment (tomkerkhove) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: I wanted to show a complete setup but how it is being used in the function is less relevant?

Comment (stijnmoreels) [reply docs/preview/guidance/use-with-dotnet-and-functions.md:32]: It's okay for me, I'll resolve this. A complete setup exam will be ok!

**Pull request merged**